testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -244,3 +247,104 @@ def test_PDB_metals():
     assert u.atoms[1].mass == pytest.approx(tables.masses["FE"])
     assert u.atoms[2].mass == pytest.approx(tables.masses["CA"])
     assert u.atoms[3].mass == pytest.approx(tables.masses["MG"])
+
+
+# An example of ideal PDB file used to test elements attribute.
```

Comment:
Rather than putting this as a comment, could you put it in the REMARK section of the PDB? (see `PDB_hex` above).